src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -257,6 +257,9 @@ private function fetchUrl($url, $locale, $limit, $region)
             if (isset($result->formatted_address)) {
                 $address = $address->withFormattedAddress($result->formatted_address);
             }
+            if (property_exists($builder, 'subpremise')) {
```

Comment:
It seams like you are checking the $builder if the "subpremise" exists. Shouldn't we check the result from google?